packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -66,23 +64,59 @@ type DefaultProps = {
     isPlaceholder: SelectOpenerProps["isPlaceholder"];
 };
 
+type SelectOpenerState = {
+    pressed: boolean;
```

Comment:
**suggestion:** It would be valuable adding a comment explaining the context behind this decision (or why we only store state for pressed but not for the other states).

Reply:
Good idea - updated!